readme.md

```diff
@@ -20,9 +20,13 @@ This plugin adds support for the [Accelerated Mobile Pages](https://www.ampproje
 
 With the plugin active, all posts on your site will have dynamically generated AMP-compatible versions, accessible by appending `/amp/` to the end your post URLs. For example, if your post URL is `http://example.com/2016/01/01/amp-on/`, you can access the AMP version at `http://example.com/2016/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
```

Comment:
Maybe `With the plugin active` became necessary from support topics where people didn't have the plugin active. But this seems unnecessary.

Reply:
From the [wiki](https://github.com/Automattic/amp-wp/wiki/General-FAQ's-and-Common-Issues#help-my-amp-url-redirects-me-to-my-non-amp-page), it look like there have been issues with users not activating the plugin. But would this message prevent that?

Reply:
Also, could we cut "dynamically generated"?